conf/broker.conf

```diff
@@ -1171,6 +1171,9 @@ defaultNamespaceBundleSplitAlgorithm=range_equally_divide
 # load shedding strategy, support OverloadShedder and ThresholdShedder, default is ThresholdShedder since 2.10.0
 loadBalancerLoadSheddingStrategy=org.apache.pulsar.broker.loadbalance.impl.ThresholdShedder
 
+# load balance placement strategy, support LeastLongTermMessageRate and LeastResourceUsageWithWeight
+loadBalancerLoadPlacementStrategy=org.apache.pulsar.broker.loadbalance.impl.LeastLongTermMessageRate
```

Comment:
To me, `LeastResourceUsageWithWeight` looks better than `LeastLongTermMessageRate`. What is our plan to make `LeastResourceUsageWithWeight` as the default?

Reply:
I don't think so. The new strategy requires more practice in the product environment before it becomes the default.